harper-core/tests/text/linters/Alice's Adventures in Wonderland.snap.yml

```diff
@@ -1523,6 +1523,15 @@ Suggest:
 
 
 
+Lint:    Punctuation (31 priority)
+Message: |
+    1304 | I’m angry. Therefore I’m mad.”
+         |            ^~~~~~~~~ Discourse markers at the beginning of a sentence should be followed by a comma.
```

Comment:
This looks like a false positive but I'm not an expert at discourse markers. It looks like good English to me. Here's some forum topics on this that I haven't read yet:
- https://english.stackexchange.com/questions/26498/should-there-always-be-a-comma-after-therefore-however-etc
- https://www.quora.com/Is-therefore-always-followed-by-a-comma
- https://www.bachelorprint.com/commas/comma-after-therefore/
- https://quillbot.com/blog/frequently-asked-questions/do-you-put-a-comma-before-or-after-therefore/